orderbook/src/fee.rs

```diff
@@ -177,6 +193,7 @@ impl MinFeeCalculator {
         }
     }
 
+    /// Computes unsubsidized min fee.
     async fn compute_min_fee(
```

Comment:
ubernit: Maybe rename the methods to avoid confusion:
- `compute_unsubsidized_min_fee`
- `get_unsubsidized_min_fee`